src/provider/controller/SelectorHandler.ts

```diff
@@ -71,6 +72,9 @@ export class SelectorHandler extends AsyncInit {
     @inject(Inject.APP_DIRECTORY)
     private _directory!: AppDirectory;
 
+    @inject(Inject.MODEL)
```

Comment:
Suggest move this class to using constructor injection, rather than adding more parameter injection. At the moment it's a bit of a mix, and I'd like it if we could move towards constructor, which I think is what `injectify` recommend

Reply:
✔ Noted as a nice to have tidy-up task